src/pages/community/members.js

```diff
@@ -201,14 +201,27 @@ const options = [
 }));
 
 const activeMember = {
-  label: "Active",
+  label: (
+    <DropdownWrapper>
+      <div className="allOptions">Active</div>
+    </DropdownWrapper>
+  ),
   value: "active",
   color: theme.linkColor,
   isFixed: true,
   icon: activeIcon,
   className: "allOptions",
 };
 
+const allMembers = {
+  label: "All Members",
+  value: "all",
+  color: theme.linkColor,
+  isFixed: true,
+  icon: null,
+  className: "allOptions",
+};
+
 const MembersPage = () => {
   /**
    * state storing the currently selected categories.
```

Comment:
At line: 246, pass a new prop named `defaultOption={activeMember}`, and use that prop in the `Dropdown` component for the `defaultValue` prop.